src/Settings/BankingInformationSettings/BankingInformationSettings.js

```diff
@@ -0,0 +1,98 @@
+import { FormattedMessage } from 'react-intl';
+import { Field, Form } from 'react-final-form';
+import { useQueryClient } from 'react-query';
+
+import {
+  Button,
+  Col,
+  Headline,
+  Loading,
+  Pane,
+  RadioButton,
+  RadioButtonGroup,
+  Row,
+} from '@folio/stripes/components';
+import { useShowCallout } from '@folio/stripes-acq-components';
+import {
+  useOkapiKy,
+  useNamespace,
+} from '@folio/stripes/core';
+
+import { useBankingInformation } from '../hooks';
+import { SETTINGS_API } from '../constants';
+
+const BankingInformationSettings = () => {
+  const { enabled, key, id: bankingInformationId, isLoading } = useBankingInformation();
+  const ky = useOkapiKy();
+  const queryClient = useQueryClient();
+  const [namespace] = useNamespace({ key: 'banking-information-settings' });
+  const sendCallout = useShowCallout();
+
+  const onSubmit = async ({ value }) => {
+    try {
+      await ky.put(`${SETTINGS_API}/${bankingInformationId}`, {
+        json: { value, key },
+      });
+      queryClient.invalidateQueries([namespace]);
```

Comment:
Probably it makes sense to use `refetch` from `useBankingInformation`: hook `useQuery` returns that fn.